permissions/sailfish-browser.profile

```diff
@@ -11,6 +11,7 @@
 ### APPLICATION
 dbus-user.own org.sailfishos.browser
 dbus-user.own org.sailfishos.browser.ui
+dbus-user.talk com.jolla.settings
```

Comment:
If there is some specific interface, path or both that browser calls, I would specify also that here with a dbus-user.call rule. Since it obviously wasn't using com.jolla.settings for anything else, it should be safe to add one.